src/components/StudyList/StudyTable.js

```diff
@@ -120,18 +158,23 @@ const StudyTable = ({
       })}
       columns={columns.columns.filter(col => col.visible).map(col => col.name)}
     >
-      <Table
-        striped
-        sortable
-        celled
-        headerRow={header}
-        tableData={
-          columns.sorting.direction === 'ascending'
-            ? studies
-            : studies.reverse()
-        }
-        renderBodyRow={data => renderRow(data, visibleCols)}
-      />
+      {({logEvent}) => (
+        <Table
+          striped
+          sortable
+          celled
+          headerRow={header}
+          tableData={
+            sorting.direction === 'ascending'
+              ? paginatedList
+              : paginatedList.reverse()
```

Comment:
The `reverse()` needs to be done before we slice the array or else only the current page will be sorted.